src/script/Container/runContainer.sh

```diff
@@ -124,5 +124,8 @@ if [ $? -eq 0 ]; then
   rm -f $tmpfile
 fi
 
+docker run -it --rm -v /data/container/:/data/hostdisk:ro --net=host \
+  registry.cern.ch/cmscrab/crabtwfilebeat:v3.latest
```

Comment:
- `--name filebeat` ? otherwise you will have a new container every time you run this script. But beware script exit 1 because container name already exist.
- Have you considered deploy with puppet?

Reply:
https://forge.puppet.com/modules/puppetlabs/docker/readme#containers apparently puppet can issue a `docker run`, very nice idea man! :)
i removed these changes from runcontainer.sh :)

Reply:
maaaan this was not hard at all :) 

https://gitlab.cern.ch/ai/it-puppet-hostgroup-vocmsglidein/-/commit/1d2ee5eecf608d97721e328ec2f5317c9c26ad3a

already running on crab-preprod-tw01